src/exampleTypes/ResultTable/Header.js

```diff
@@ -27,6 +27,14 @@ const moveColumn = (
   })
 }
 
+const addColumn = (mutate, computeSideIndex, includes, triggerField, field) => {
```

Comment:
Why not just inline this function? It takes 5 args and it's used only once

Reply:
It could be sure. Will inline it since it does not seem like we would reuse this etc.